README.md

```diff
@@ -69,15 +69,15 @@ $ ppm --help
 # change minimum-stability to dev in your composer.json (until we have a version tagged): "minimum-stability": "dev"
 composer require php-pm/php-pm:dev-master
 composer require php-pm/httpkernel-adapter:dev-master #if you have httpkernel (laravel, symfony)
-./vendor/bin/ppm config --bootstrap=symfony #places a ppm.json in your directory
-./vendor/bin/ppm start #reads ppm.json and starts the server like you want
+./bin/ppm config --bootstrap=symfony #places a ppm.json in your directory
+./bin/ppm start #reads ppm.json and starts the server like you want
 ```
 
 Once configured (composer and ppm.json) you can start your app on your development machine or server instantly:
 
 ```bash
 composer install
-./vendor/bin/ppm start
+./bin/ppm start
```

Comment:
All binaries are placed in `vendor/` in default composer, actually only if your reconfigure it its somewhere else (like Symfony Standard edition it does. So I'd like to keep it with `vendor/`. WDYT?